<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paintToolBar</class>
 <widget class="QWidget" name="paintToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>194</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>194</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>工具</string>
  </property>
  <widget class="QSlider" name="tipSlider">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>50</y>
     <width>22</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="tipSpinBox">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>220</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>95</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="penTypeBox">
      <property name="currentText">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>类型</string>
      </property>
      <item>
       <property name="text">
        <string>画笔</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>橡皮擦</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColorSelect">
      <property name="text">
       <string>选取颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnZoomIn">
      <property name="text">
       <string>放大图像</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnZoomOut">
      <property name="text">
       <string>缩小图像</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColorize">
      <property name="text">
       <string>开始上色</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>笔触宽度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
